### DISCUSSION TITLE
Options to update configuration in renovate.json in all onboarded repos

### DISCUSSION THREAD

**Discussion by divlaks** [renovatebot/renovate#15599] [Request Help] [answered]
Hi Team,

We are looking at options to update configuration in renovate.json file in all the onboarded repos. Is there any option to do the same or do we have to update renovate.json manually in each repo.

Thanks,
Dhivya

**Comment by viceice** [comment, marked as answer]
You need to manually do it. Use shared presets, so you only need to update the preset in future.

https://docs.renovatebot.com/config-presets/

**Reply by divlaks** [reply]
Thanks!

**Reply by rarkins** [reply]
Check out https://github.com/octoherd for bulk repository updating scripts
